### PR TITLE
add demo gif - refs #253277

### PR DESCRIPTION
**Pull request by tedw87**

### CODE REVIEW

**Review by tiberiuichim — COMMENTED** [review]
The reason you don't see results when you initially save is because you need to set the result item card to something other then "Items". Compact listing, horizontal cards, they work better.

**Comment by tedw87** [comment]
> The reason you don't see results when you initially save is because you need to set the result item card to something other then "Items". Compact listing, horizontal cards, they work better.

I understand, thank you for the suggestions.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 ## Features
 
+![Searchlib](./docs/volto-searchlib.gif)
```
Comment (avoinea) [inline README.md:21]: @tedw87 Please use the absolute URL, e.g. https://github.com/eea/volto-redmine-helpdesk/pull/15/files

**Review by avoinea — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tedw87) [reply README.md]: Done

**Review by avoinea — APPROVED** [review]

**Pull request merged**